test/resources/bookshop-graphql/package.json

```diff
@@ -14,6 +14,9 @@
         "kind": "mocked"
       }
     },
+    "features": {
```

Comment:
which feature flag is this?

Reply:
it enables use of in-memory db in production

Reply:
ok, didn't see that this is for production tests.